Dockerfile.scratch

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.14-alpine as builder
```

Comment:
How about making this build the default?

Reply:
Sorry for the huge delay, been a bit busy too. It might be good to publish scratch and alpine containers, so the latter could be used for debugging?